server/monitor/src/main/java/org/apache/accumulo/monitor/next/Endpoints.java

```diff
@@ -57,6 +57,15 @@
 
 @Path("/")
 public class Endpoints {
+  /**
+   * A {@code String} constant representing supplied resource group
+   */
+  private static final String GROUP = "group";
+
+  /**
+   * A {@code String} constant representing supplied tableId.
+   */
+  private static final String TABLEID = "tableId";
```

Comment:
```suggestion
  /**
   * A {@code String} constant representing supplied resource group
   */
  private static final String GROUP = "group";

  /**
   * A {@code String} constant representing supplied tableId.
   */
  private static final String TABLEID = "tableId";
```
I think it might be better to update these to make it more evident that these are the parameter keys. Maybe something like `GROUP_PARAM_KEY` AND `TABLEID_PARAM_KEY`. The javadoc should be updated to reflect this if the change is made.

Reply:
It does make sense!